whitepapers/amount.md

```diff
@@ -41,6 +38,12 @@ For an in-depth guide to sharing feedback, visit our complete [How to Contribute
   </tr>
   </table>
 
+## Response to Public Feedback (August 2015)
+
+Data elements related to amount are crucial to Federal spending transparency and are important to many stakeholder groups, within and outside of the Federal Government. During the three week feedback period, 3 individuals and organizations provided feedback through GitHub, which is available here, and OMB and Treasury received feedback from the Government Accountability Office (GAO). The majority of this input brought up questions about the context of these amounts (including which data elements are cumulative), the applicability of these data elements to procurement and financial assistance, and the sources of this information.
```

Comment:
When we launched this site, we decided not to lower-case Federal (unless it's part of an acronym like _Federal Funding Accountability and Transparency Act). IIRC, the Treasury and OMB style guides differ on this, but we decided to roll with the less intimidating lower case.